rfcs/proposed/numa_support/increase-numa-support-availability.org

```diff
@@ -0,0 +1,115 @@
+# -*- fill-column: 80; -*-
+
+#+title: Improve predictability of API for NUMA support
+
+*Note:* This is a sub-RFC of the https://github.com/oneapi-src/oneTBB/pull/1535.
+Specifically, its section about "Increased availability of NUMA support".
+
+* Introduction
+oneTBB has a soft dependency on several variants of ~tbbbind~, which are loaded
+by the library as part of its initialization stage. In turn, each ~tbbbind~ has
+a hard dependency, i.e., relies on load-time linking, on a concrete version of
+the HWLOC library [1, 2]. The soft dependency of oneTBB on ~tbbbind~ allows the
+library to continue its execution even if the system loader is unable to resolve
+the hard dependency on HWLOC for ~tbbbind~. In this case, the HW topology is not
+discovered and the machine is seen as if all CPU cores were uniform, which is
+the default TBB behavior when NUMA constraints are not used. Thus, the following
+code returns meaningless values as these values are just ignored by oneTBB:
```

Comment:
Would it be better to say that the returned values do not reflect the real topology of the machine? I am not sure in which sense these values are ignored. If you use these to create constrained arenas (well, a single arena), they are not formally ignored, rather they do not really matter.